sample_openapi/param-collision.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
where is this file used?

Reply:
oops - somehow missed the important part where we invoke a test that uses it.. that's fixed now.